src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -147,7 +147,7 @@ export default class FilingHistoryList extends Mixins(FilingMixin) {
       case EnumUtilities.isTypeAmalgamation(filing): return 'amalgamation-filing'
       case EnumUtilities.isTypeChangeOfAddress(filing): return 'change-of-address'
       case EnumUtilities.isTypeConsentContinuationOut(filing): return 'consent-continuation-out'
-      case EnumUtilities.isTypeContinuationInApplication(filing): return 'continuation-in-application'
+      case EnumUtilities.isTypeContinuationIn(filing): return 'continuation-in'
```

Comment:
Just trying to be consistent with the other filing names (several of which are "applications" but none have that in their name).

Reply:
Hmmm, I'm thinking in this case, it's kind of similar to Incorporation Application? We can also see it in the title.

Reply:
I'm just wondering, no need to change it back Sev.

Reply:
Yes, it's a "Continuation In Application".

But, also, there is "Amalgamation Application", but we don't call it that.